### PR TITLE
V2

### PR DESCRIPTION
**Pull request by cdworak**

### CODE REVIEW

File: error.go [hunk 1]
```diff
@@ -62,3 +62,18 @@ func (e Errors) Warnings() []Error {
 	}
 	return warnings
 }
+
+func IsNotFoundError(err error) bool {
+	if e, ok := err.(Errors); ok {
+		for _, innerError := range e {
+			if IsNotFoundError(innerError) {
```
Comment (mhupman) [inline error.go]: Didn't we find this needed to be a pointer?

File: error_test.go [hunk 2]
```diff
@@ -0,0 +1,56 @@
+package yext
+
+import "testing"
+
+func TestIsNotFoundError(t *testing.T) {
```
Comment (mhupman) [inline error_test.go]: yay!

File: location_diff.go [hunk 3]
```diff
@@ -22,7 +26,8 @@ import "reflect"
 //   // isDiff -> false
 //   // delta -> nil
 func (y Location) Diff(b *Location) (d *Location, diff bool) {
-	// TODO(bjm): remove this once the ETL is upgraded to use hydration
+	// TODO (bjm):
+	// remove this once the ETL is upgraded to use hydration
```
Comment (mhupman) [inline location_diff.go]: why the newlines?

File: user_diff_test.go [hunk 4]
```diff
@@ -3,7 +3,7 @@ package yext_test
 import (
 	"testing"
 
-	"github.com/yext/yext-go"
+	yext "gopkg.in/yext/yext-go.v2"
```
Comment (mhupman) [inline user_diff_test.go]: Shouldn't need to import at all since we're in the same package...

File: error.go (hunk 1 above)
Comment (cdworak) [inline error.go]: having it be a pointer was what was causing it to both skip the  if (Errors), else if (Error) when we passed the single innerError. When I removed the pointer the tests all pass. Adding the pointer in fails the single Error tests

Comment (mhupman) [inline error.go]: Oh, ok.  Was confused.

File: user_diff_test.go (hunk 4 above)
Comment (cdworak) [inline user_diff_test.go]: it's in package yext_test...i'm not sure why considering all the other tests (ex: customfield_test) are in package yext. i'll move to package yext unless there's any reason not to

**Pull request closed without merging**